app/javascript/src/Types/swagger.d.ts

```diff
@@ -0,0 +1,3 @@
+declare module 'swagger-client/es/execute' {
+  export function execute (req: unknown): unknown
+}
```

Comment:
```suggestion
// Reference:
// - https://github.com/swagger-api/swagger-js/blob/master/src/index.js
// - https://github.com/swagger-api/swagger-js/blob/master/src/execute/index.js
declare module 'swagger-client' {
  import type { SupportedHTTPMethods } from 'swagger-ui'

  export interface ExecuteData {
    contextUrl: string;
    fetch: (arg: unknown) => unknown;
    method: SupportedHTTPMethods;
    operation: unknown;
    operationId: string;
    parameters: unknown;
    pathName: string;
    requestBody?: unknown;
    requestContentType: string;
    requestInterceptor?: ((request: Request) => Promise<Request> | Request) | undefined;
    responseContentType: string;
    responseInterceptor?: ((response: Response) => Promise<Response> | Response) | undefined;
    scheme: string;
    securities: unknown;
    server: string;
    serverVariables: unknown;
    spec: unknown;
  }
  function execute (req: ExecuteData): unknown
  export { execute }
}
```

Reply:
This doesn't work for me :shrug: 

Getting this error:

```
ERROR in ./app/javascript/src/ActiveDocs/ThreeScaleApiDocs.ts 120:15-23
"export 'execute' (imported as '_execute') was not found in 'swagger-client'
```
![Screenshot from 2023-04-12 14-00-47](https://user-images.githubusercontent.com/1270328/231450841-ab61693b-7462-4396-bd59-a5a88553ad03.png)

I would keep it as it is right now.

Reply:
I'll retry again, did you update the import to `import { execute } from 'swagger-client'`?

> I would keep it as it is right now.

If the types belong to a 3rd party library I would strongly recommend to move them into a `d.ts` file and import them from the pacakge, not from `Types/`.

Reply:
> did you update the import to `import { execute } from 'swagger-client'`?

Yes, I did.

I think the problem is that it needs to be consistent with the actual library code, and this is not how `execute` is actually exported in `swagger-client`, but I'm not sure if that's the reason.